test/functional/wallet_multiwallet.py

```diff
@@ -151,8 +151,8 @@ def wallet_file(name):
         competing_wallet_dir = os.path.join(self.options.tmpdir, 'competing_walletdir')
         os.mkdir(competing_wallet_dir)
         self.restart_node(0, ['-walletdir=' + competing_wallet_dir])
-        exp_stderr = r"Error: Error initializing wallet database environment \"\S+competing_walletdir\"!"
-        self.nodes[1].assert_start_raises_init_error(['-walletdir=' + competing_wallet_dir], exp_stderr, match=ErrorMatch.PARTIAL_REGEX)
+        exp_stderr = "Error: Cannot obtain a lock on wallet directory {0}. {1} is probably already running.".format(competing_wallet_dir, self.config['environment']['PACKAGE_NAME'])
+        self.nodes[1].assert_start_raises_init_error(['-walletdir=' + competing_wallet_dir], exp_stderr)
```

Comment:
Would also be good to temporarily enable the functional tests on appveyor again to see if the full path works on windows as well

Reply:
Done: https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/35614117

Reply:
```
                                   AssertionError: [node 1] Expected message 
"Error: Cannot obtain a lock on wallet directory C:\Users\appveyor\AppData\Local\Temp\1\test_runner_₿_🏃_20201007_132949\wallet_multiwallet_117\competing_walletdir. Bitcoin Core is probably already running." does not fully match stderr:
                                   
"Error: Cannot obtain a lock on wallet directory C:/Users\appveyor\AppData\Local\Temp\1\test_runner_₿_🏃_20201007_132949\wallet_multiwallet_117\competing_walletdir. Bitcoin Core is probably already running."

Reply:
Lol that's fun ^

Reply:
[Fixed](https://github.com/bitcoin/bitcoin/pull/20094#issuecomment-706560784).

https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/35676234: `wallet_multiwallet.py` and `feature_filelock.py` passed.

@MarcoFalke 
Could the _"[DO NOT MERGE] ci: Temporarily enable the functional tests on AppVeyor"_ commit be dropped?